### PR TITLE
Use color for Button component instead of type

### PR DESCRIPTION
**Pull request by ikeyu0806**
### What changed?
 - Use `color` for Button component instead of `type`
 - Closes https://github.com/code4romania/taskforce-fe-components/issues/149

### CODE REVIEW

**Comment by cipick** [comment]
We should keep the `type` prop, but accept the following values:
`type: PropTypes.oneOf(["button", "submit","reset"]),`

File: src/components/button/button.js [hunk 1]
```diff
@@ -40,15 +40,15 @@ export const Button = ({
 Button.propTypes = {
   children: PropTypes.node.isRequired,
   onClick: PropTypes.func.isRequired,
-  type: PropTypes.oneOf(["primary", "danger"]),
+  color: PropTypes.oneOf(["primary", "danger"]),
   size: PropTypes.oneOf(["small", "medium", "large"]),
   disabled: PropTypes.bool,
   inverted: PropTypes.bool,
   inputType: PropTypes.oneOf(["button", "submit", "reset"])
```
Comment (surdu) [inline src/components/button/button.js:47]: Please also add a `type` prop to be used as the button's type

**Comment by surdu** [comment]
@ikeyu0806 Please also increase the package version to 2.0.0, as this introduces braking changes.

We might want to postpone this PR until we gather more items that introduce braking changes and until all taskforce projects release their MVPs.

File: src/components/button/button.js (hunk 1 above)
Comment (ikeyu0806) [reply src/components/button/button.js:47]: Thank you for the review!

This component already has a prop called `inputType`. Should I rename it to ｀type｀?

Comment (surdu) [reply src/components/button/button.js:47]: Yes, I believe renaming it will make it more intuitive.

Comment (ikeyu0806) [reply src/components/button/button.js:47]: I will fix it in that way!

**Comment by ikeyu0806** [comment]
@surdu 
The following fixes have been made.

* rename `inputType` prop to `type`.
* increase the package version to 2.0.0.

And I confirmed.
> We might want to postpone this PR until we gather more items that introduce braking changes and until all taskforce projects release their MVPs.

**Comment by surdu** [comment]
Looks good 🚀 ! Thanks for the help ! I won't approve it yet, not to be accidentally merged until we have more things for v2.

**Comment by ikeyu0806** [comment]
@surdu 
Thank you for the review😄

**Comment by surdu** [comment]
@ikeyu0806 We plan to release a new major version and it will be the perfect time to merge this. Do you have time to please take care of the conflicts for this PR ?

**Comment by ikeyu0806** [comment]
@surdu 
Thank you for contacting.

Please check as conflict has been dealt with.

Read this commit. And add warning type, make onClick callback optional.
https://github.com/code4romania/taskforce-fe-components/commit/244c781305c68f45d96f6e250c879fba90cb0c7e

**Pull request merged**